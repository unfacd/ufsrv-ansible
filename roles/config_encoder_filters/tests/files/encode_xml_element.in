element:
  - element1:
    - element2: ''
    - element3: some text
