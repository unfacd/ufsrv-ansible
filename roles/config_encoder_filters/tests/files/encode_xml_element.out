<element>
  <element1>
    <element2 />
    <element3>some text</element3>
  </element1>
</element>
